### PR TITLE
Modify PowerPoint GitHub examples repository for document processing playground

### PR DESCRIPTION
**Pull request by DharanitharanA**
**Description:**
Changed below GitHub examples for play ground button
1. Security
2. SmartArt
3. Macros
4. PowerPoint presentation
5. Slides
6. Shapes
7. Tables
8. Sections

### CODE REVIEW

File: Macros/Load-and-save-macro-PowerPoint/.NET/Load-and-save-macro-PowerPoint/Load-and-save-macro-PowerPoint.csproj
Comment (LokeshBaskar) [inline Macros/Load-and-save-macro-PowerPoint/.NET/Load-and-save-macro-PowerPoint/Load-and-save-macro-PowerPoint.csproj:1]: Please remove the macro file changes it is already changes done.

File: Shapes/Create-PowerPoint-groupshape/.NET/Create-PowerPoint-groupshape/Program.cs [hunk 1]
```diff
@@ -9,11 +9,11 @@
 //Add a TextBox to the group shape.
 groupShape.Shapes.AddTextBox(30, 25, 100, 100).TextBody.AddParagraph("My TextBox");
 //Get a picture as stream.
-using FileStream pictureStream = new FileStream(@"../../../Data/Image.jpg", FileMode.Open);
+using FileStream pictureStream = new FileStream(@"Data/Image.jpg", FileMode.Open);
```
Comment (LokeshBaskar) [inline Shapes/Create-PowerPoint-groupshape/.NET/Create-PowerPoint-groupshape/Program.cs:12]: Please use the Path.GetFullPath

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]
Please address the feedbacks

File: Shapes/Create-PowerPoint-groupshape/.NET/Create-PowerPoint-groupshape/Program.cs (hunk 1 above)
Comment (DharanitharanA) [reply Shapes/Create-PowerPoint-groupshape/.NET/Create-PowerPoint-groupshape/Program.cs]: Added properly

File: Macros/Load-and-save-macro-PowerPoint/.NET/Load-and-save-macro-PowerPoint/Load-and-save-macro-PowerPoint.csproj
Comment (DharanitharanA) [reply Macros/Load-and-save-macro-PowerPoint/.NET/Load-and-save-macro-PowerPoint/Load-and-save-macro-PowerPoint.csproj:1]: Removed the changes

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**